rules/scala.bzl

```diff
@@ -14,21 +17,30 @@ load(
     _ZincConfiguration = "ZincConfiguration",
     _ZincInfo = "ZincInfo",
 )
+
+# This line is unneeded, as phase_zinc_compile is already wrapped inside ext_zinc_compile (imported below)
```

Comment:
If it's unused, let's remove it with a separate PR, rather than documenting it this way.